data/sql/updates/pending_db_world/rev_1598238244990167657.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598238244990167657');
+
+UPDATE `creature_template` SET `faction` = 21 WHERE `entry` = 14688;
```

Comment:
We need to verify that this change is correct and do not break other stuff

Reply:
double checked the faction here:
https://wowgaming.altervista.org/aowow/?npc=14688

Looked up the factiontemplate:
https://www.azerothcore.org/wiki/FactionTemplate

21 looks to be correct. 
Anything else i should check?